src/routes/exception.ts

```diff
@@ -19,32 +19,59 @@
 
 import * as express from 'express';
 import multer from 'multer';
-import exceptionApi, {
+import propertyExceptionApi, {
 	requestContainsFile,
 } from '../exception/property-exceptions/property-exception-api';
 import { wrapAsync } from '../middleware';
 import featureFlags from '../feature-flags';
 import { Request, Response } from 'express';
 import { ExceptionErrorHandler } from '../exception/property-exceptions/error-handling';
+import missingEntityExceptionApi from '../exception/missing-entity-exceptions/api';
 
-// config
-const router = express.Router({ mergeParams: true });
+/**
+ * Property exceptions
+ */
+const propertyExceptionRouter = express.Router({ mergeParams: true });
 const upload = multer({ dest: '/tmp' });
 
-// routes
-router.post('*', upload.single('exceptionFile'), requestContainsFile);
-router.post('/', wrapAsync(exceptionApi.createProgramException));
-router.post('/entity', wrapAsync(exceptionApi.createEntityException));
+// Middleware
+propertyExceptionRouter.use(ExceptionErrorHandler);
+
+// GET
+propertyExceptionRouter.get('/', wrapAsync(propertyExceptionApi.getProgramException));
+propertyExceptionRouter.get('/entity', wrapAsync(propertyExceptionApi.getEntityException));
+
+// POST
+propertyExceptionRouter.post('*', upload.single('exceptionFile'), requestContainsFile);
+propertyExceptionRouter.post('/', wrapAsync(propertyExceptionApi.createProgramException));
+propertyExceptionRouter.post('/entity', wrapAsync(propertyExceptionApi.createEntityException));
+
+// DELETE
+propertyExceptionRouter.delete('/', wrapAsync(propertyExceptionApi.clearProgramException));
+propertyExceptionRouter.delete(
+	'/entity/:entity',
+	wrapAsync(propertyExceptionApi.deleteEntityException),
+);
+
+/**
+ * Missing entity exceptions
+ */
+const missingEntityExceptionRouter = express.Router({ mergeParams: true });
 
-router.get('/', wrapAsync(exceptionApi.getProgramException));
-router.get('/entity', wrapAsync(exceptionApi.getEntityException));
+// GET
+missingEntityExceptionRouter.get('/', (req, resp) => resp.send({ msg: 'not implemented yet...' }));
 
-router.delete('/', wrapAsync(exceptionApi.clearProgramException));
+// POST
+missingEntityExceptionRouter.post('/', wrapAsync(missingEntityExceptionApi.createEntityException));
 
-router.delete('/entity/:entity', wrapAsync(exceptionApi.deleteEntityException));
+const notFound = (req: Request, res: Response) => res.status(404).send();
 
-router.use(ExceptionErrorHandler);
+// Parent
+const exceptionRouter = express.Router({ mergeParams: true });
+exceptionRouter.use(
+	'/property-exception',
+	featureFlags.FEATURE_SUBMISSION_EXCEPTIONS_ENABLED ? propertyExceptionRouter : notFound,
+);
+exceptionRouter.use('/missing-entity-exception', missingEntityExceptionRouter);
```

Comment:
Exceptions are in place now and won't be disabled, let's remove the feature flag entirely. Takes away the need for `notFound` endpoints.

Reply:
https://github.com/icgc-argo/argo-clinical/pull/1110/commits/39551ffbda67286a4e8241af094246aeb583453e